### DISCUSSION TITLE
Collect transcriptions for meeting's summary

### DISCUSSION THREAD

**Discussion by lovedota** [Azure/communication-ui-library#4778] [Q&A]
Hi Team

We are using MsTeam for our internal meeting, in order to use the caption function, users need to enable Caption in the UI.

![Screen Shot 2024-06-02 at 07 37 20 AM](https://github.com/Azure/communication-ui-library/assets/417914/b98375bf-96a0-48dd-bfe9-f2a3f6fe5d4f)

```
adapter.on('captionsReceived', (e) => {
  console.log('Caption Recieved Event:', e);
});
```

Do we have another way to turn it programmatically, it will be used in our internal company meeting and we want to use AI to summary the transcription?

**Comment by carocao-msft** [comment]
Hi sorry for the late response, but yes we do! We have updated our storybook documentation to include code snippets for starting captions in the background, and collecting captions data through listening to captionsReceived event. https://azure.github.io/communication-ui-library/?path=/docs/concepts-closed-captions--docs
